doc/code_snippets/snippets/connectors/go/hello.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/tarantool/go-tarantool/v2"
+	"time"
+)
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+	dialer := tarantool.NetDialer{
+		Address:  "127.0.0.1:3301",
+		User:     "sampleuser",
+		Password: "123456",
+	}
+	opts := tarantool.Opts{
+		Timeout: time.Second,
+	}
+
+	conn, err := tarantool.Connect(ctx, dialer, opts)
+	if err != nil {
+		fmt.Println("Connection refused:", err)
+		return
+	}
+
+	// Interacting with the database
+	// Insert data
+	tuple1, err := conn.Do(
+		tarantool.NewInsertRequest("bands").
+			Tuple([]interface{}{1, "Roxette", 1986}),
+	).Get()
+	tuple2, err := conn.Do(
+		tarantool.NewInsertRequest("bands").
+			Tuple([]interface{}{2, "Scorpions", 1965}),
+	).Get()
+	tuple3, err := conn.Do(
+		tarantool.NewInsertRequest("bands").
+			Tuple([]interface{}{3, "Ace of Base", 1987}),
+	).Get()
+	tuple4, err := conn.Do(
+		tarantool.NewInsertRequest("bands").
+			Tuple([]interface{}{4, "The Beatles", 1960}),
+	).Get()
+	fmt.Println("Inserted tuples: ", tuple1, tuple2, tuple3, tuple4)
+
+	// Select by primary key
+	data, err := conn.Do(
+		tarantool.NewSelectRequest("bands").
+			Limit(10).
+			Iterator(tarantool.IterEq).
+			Key([]interface{}{uint(1)}),
+	).Get()
+	fmt.Println("Tuple selected the primary key value:", data)
```

Comment:
It is recommended to check the `err` in Go every time.
```suggestion
	).Get()
	if err != nil {
	    fmt.Println("Select error:", err)
	}
	fmt.Println("Tuple selected the primary key value:", data)
```

Reply:
Added error checks for all data operations